linux/flatpak/org.jacktrip.JackTrip.Devel.yml.j2

```diff
@@ -0,0 +1,29 @@
+app-id: org.jacktrip.JackTrip.Devel
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: jacktrip
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  - --device=dri
+  # Wayland access
+  - --socket=wayland
+  # Needs network access
+  - --share=network
+  # Pipewire/Jack
+  - --filesystem=xdg-run/pipewire-0
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+  - --socket=system-bus
+modules:
+  - shared-modules/linux-audio/jack2.json
+  - name: jacktrip
+    buildsystem: meson
+    config-opts:
+      - -Dprofile=development
```

Comment:
My question was what does this flag do to the build? Could this be documented somewhere?

Reply:
It's setting the app id to org.jacktrip.JackTrip.Devel instead of org.jacktrip.JackTrip, so that you can install the stable and dev versions side by side. This should probably be documented somehow.

Reply:
I was asking about `-Dprofile=development`, isn't that a build/meson setting?

Reply:
Yes. That's what I'm talking about. It's called profile because it's common in other projects. And I don't like the idea of setting the app id as a string. What's confusing you about it?

Reply:
Sorry, maybe I should've looked into meson source instead of asking you - after looking there I understand what's happening. The connection between `-Dprofile=development` and application id is definitely unclear unless you know what's in `meson.build`.

In case anyone else is looking at this: my original question was what does the "development" profile change in the build system. The answer is that it changes `application_id`, used in the installation step in the Linux desktop integration files; it does _not_ change anything in the cpp build itself, i.e. does not trigger a debug build.

Reply:
No problem. Generally, if you don't know what configure options do, you can just type `meson configure` in your build directory:
```
  Project options                          Current Value                              Possible Values                            Description
  ---------------                          -------------                              ---------------                            -----------
  jack                                     auto                                       [enabled, disabled, auto]                  Build with JACK Backend
  nogui                                    false                                      [true, false]                              Build without graphical user interface
  profile                                  default                                    [default, development]                     Choose build profile / Sets desktop id accordingly
  rtaudio                                  enabled                                    [enabled, disabled, auto]                  Build with RtAudio Backend
  wair                                     false                                      [true, false]                              WAIR
  weakjack                                 false                                      [true, false]                              Weak link JACK library
```

So maybe the description for profile could be improved. What do you think?